sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.institutions.controller.validateOffering.e2e-spec.ts

```diff
@@ -55,77 +60,42 @@ describe("EducationProgramOfferingInstitutionsController(e2e)-validateOffering",
       InstitutionTokenTypes.CollegeFUser,
       collegeFLocation,
     );
-    const fakeEducationProgram = createFakeEducationProgram({
+    const fakeFullTimeEducationProgram = createFakeEducationProgram({
       institution: collegeF,
       user: collegeFUser,
     });
-    fakeEducationProgram.sabcCode = faker.random.alpha({ count: 4 });
-    fakeEducationProgram.deliveredOnline = true;
-    collegeFEducationProgram = await db.educationProgram.save(
-      fakeEducationProgram,
+    const fakePartTimeEducationProgram = createFakeEducationProgram(
+      {
+        institution: collegeF,
+        user: collegeFUser,
+      },
+      {
+        initialValue: {
+          programIntensity: ProgramIntensity.fullTimePartTime,
+          deliveredOnSite: true,
+        },
+      },
     );
-  });
-
-  it("Should validate an offering when passed valid data.", async () => {
-    // Arrange
-    const institutionUserToken = await getInstitutionToken(
-      InstitutionTokenTypes.CollegeFUser,
+    fakeFullTimeEducationProgram.sabcCode = faker.random.alpha({ count: 4 });
+    fakeFullTimeEducationProgram.deliveredOnline = true;
```

Comment:
Why is it not added to initial values like `fakePartTimeEducationProgram` as the existing factory supports it.